tests/examples/test_examples.py

```diff
@@ -110,3 +114,42 @@ def test_example_sample_lib():
 
             # Client worked well, let's stop the server
             server.terminate()
+
+
+def test_lazy_load_local_models_invalid_model_dir():
```

Comment:
`examples` is not a good place to add this test. I'm assuming `test_model_manager` to test `model_manager`?